### PR TITLE
doc: Add nrf54l15 entry in sample rows

### PR DESCRIPTION
**Pull request by dchat-nordic**
Add nrf54l15_cpuapp (without soc1) entry
in the sample_board_rows file.

### CODE REVIEW

File: doc/nrf/includes/sample_board_rows.txt [hunk 1]
```diff
@@ -221,3 +221,7 @@
 .. nrf54l15pdk_nrf54l15_cpuapp@soc1
 
 | :ref:`nRF54L15 PDK <ug_nrf54l15_gs>` | | ``nrf54l15pdk_nrf54l15_cpuapp`` | ``nrf54l15pdk_nrf54l15_cpuapp@soc1`` |
```
Comment (FrancescoSer) [inline doc/nrf/includes/sample_board_rows.txt:223]:
```suggestion
| :ref:`nRF54L15 PDK <ug_nrf54l15_gs>` | | ``nrf54l15pdk_nrf54l15_cpuapp`` | ``nrf54l15pdk_nrf54l15_cpuapp@soc1`` |
```

Isn't this ref going to conflict with the one in line 227? I wondering this line shouldn't be removed, as soc1 is out of scope in sdk-nrf now.

Comment (FrancescoSer) [reply doc/nrf/includes/sample_board_rows.txt:223]: @annwoj @gmarull Any insights here?

**Comment by dchat-nordic** [comment]
Similar changes merged a few minutes ago.

File: doc/nrf/includes/sample_board_rows.txt (hunk 1 above)
Comment (gmarull) [reply doc/nrf/includes/sample_board_rows.txt]: soc1 is not used upstream, so it's wrong

**Pull request closed without merging**